base/src/main/java/com/smartdevicelink/transport/TransportConstants.java

```diff
@@ -89,7 +89,6 @@
     public static final String BIND_REQUEST_TYPE_ALT_TRANSPORT = "BIND_REQUEST_TYPE_ALT_TRANSPORT";
     public static final String BIND_REQUEST_TYPE_STATUS = "BIND_REQUEST_TYPE_STATUS";
     public static final String BIND_REQUEST_TYPE_USB_PROVIDER = "BIND_REQUEST_TYPE_USB_PROVIDER";
-
```

Comment:
@kostyaBoss Please place the line `public static final String PING_ROUTER_SERVICE_EXTRA = "ping.router.service";` right below line 91.